test/integration/vaultauthmethods_integration_test.go

```diff
@@ -30,7 +30,10 @@ func TestVaultAuthMethods(t *testing.T) {
 	testK8sNamespace := "k8s-tenant-" + testID
 	testKvv2MountPath := consts.KVSecretTypeV2 + testID
 	testVaultNamespace := ""
-	k8sConfigContext := "kind-" + clusterName
+	k8sConfigContext := os.Getenv("KIND_CLUSTER_CONTEXT")
```

Comment:
Seems a bit odd that `KIND_CLUSTER_CONTEXT` is being used as any k8s config context. Perhaps we should rename that variable? Or add a boolean for KIND? 

``` if KIND; ... ```

Reply:
yeah 😅 I didn't want to hinder you guys with the new naming but ideally, it should be a generalized name? Happy to make it a part of this PR if you'd like.

Reply:
go for it!